### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kevinhammond**
Clarified build instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,10 +38,19 @@ The latest supported networks can be found at https://hydra.iohk.io/job/Cardano/
 ## Cabal
 
 #### Dependencies:
+
+You will need to install one or both of these libraries on your system:
+
 * libsodum
```
Comment (intricate) [inline README.md:44]: Can we correct this too?

```suggestion
* libsodium
```

File: README.md [hunk 2]
```diff
@@ -52,11 +61,16 @@ There are two choices for building `cardano-crypto-praos`, selected via a cabal
...
 The second option is currently only suitable for development -- but not deployment -- because it uses the libsodium C code in an unsupported configuration
 
-Use [Cabal - Version 3.0](https://www.haskell.org/cabal/) to build this project:
+Use [Cabal - Version 3.0](https://www.haskell.org/cabal/) to build this project, including the CLI interface to the node:
+
+```
+$ cabal build cardano-node cardano-cli
+```
+
+Install the `cardano-node` and `cardano-cli` commands using `cabal`.  Not that you may need to specify the installation directory using `--installdir`.
```
Comment (intricate) [inline README.md:70]:
```suggestion
Install the `cardano-node` and `cardano-cli` commands using `cabal`.  Note that you may need to specify the installation directory using `--installdir`.
```

File: README.md [hunk 3]
```diff
@@ -66,7 +80,7 @@ You can download [here](https://hydra.iohk.io/job/Cardano/cardano-node/cardano-n
 
 ### Instructions
 
-The download includes cardano-node.exe and a .dll. To run the node with cardano-node run you need to reference a few files and directories as arguments. These can be copied from the cardano-node repo into the executables directory. The command to run the node on mainnet looks like this:
+The download includes `cardano-node.exe` and a `.dll` file. To run the node with cardano-node run you need to reference a few files and directories as arguments. These can be copied from the `cardano-node` repo into the executables directory. The command to run the node on mainnet looks like this:
```
Comment (intricate) [inline README.md:83]:
```suggestion
The download includes `cardano-node.exe` and a `.dll` file. To run the node with `cardano-node run`, you need to reference a few files and directories as arguments. These can be copied from the `cardano-node` repo into the executables directory. The command to run the node on mainnet looks like this:
```

**Comment by kevinhammond** [comment]
Looks like another bors failure

**Pull request merged**